apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -7693,30 +7692,31 @@ span#HW_badge_cont:nth-child(3) {
 
 .on-setting-page .nav-left-bar.open,
 .on-user-page .nav-left-bar.open on-user-page {
-    left: -240px !important;
-    display: none !important;
+    left: 0 !important;
+    display: block !important;
 }
 .setting-sidebar-open .nav-left-bar,
 .on-user-page .nav-left-bar,
 .on-setting-page .nav-left-bar {
-    display: none;
+    display: block;
 }
 .menu_open aside-setting.setting-sidebar {
-    left: 0 !important;
+    right: 0 !important;
 }
 .on-setting-page aside-setting.setting-sidebar,
 .on-user-page aside-setting.setting-sidebar {
-    left: -250px;
+   // left: -250px;
```

Comment:
Do remove the whole style, avoid empty CSS rules in scss files

Reply:
Removed